src/core/router.go

```diff
@@ -98,6 +98,9 @@ func initRouters() {
 	beego.Router("/api/targets/ping", &api.TargetAPI{}, "post:Ping")
 	beego.Router("/api/logs", &api.LogAPI{})
 	beego.Router("/api/configs", &api.ConfigAPI{}, "get:GetInternalConfig")
+
+	beego.Router("/api/config/:group", &api.ConfigSettingAPI{}, "get:GetConfigByGroup")
+
 	beego.Router("/api/configurations", &api.ConfigAPI{})
 	beego.Router("/api/configurations/reset", &api.ConfigAPI{}, "post:Reset")
```

Comment:
remove /api/configurations/reset as it's useless.

Reply:
Removed